easybuild/easyblocks/generic/rpackage.py

```diff
@@ -215,11 +218,22 @@ def run(self):
             lib_install_prefix = os.path.join(self.installdir, self.cfg['exts_subdir'])
             mkdir(lib_install_prefix, parents=True)
 
-        if self.patches:
+        if self.patches or self.cfg['update_config_guess']:
             super(RPackage, self).run(unpack_src=True)
         else:
             super(RPackage, self).run()
 
+        if self.cfg['update_config_guess']:
+            confmake = ConfigureMake(self.cfg)
```

Comment:
IMO the better option would be to make `obtain_config_guess` a static or better free function. It mostly is already anyway.